Trello_items.py

```diff
@@ -0,0 +1,78 @@
+from secrets import KEY, TOKEN
+import requests
+
+todo_boardid = "5f6076c34c5f48265943e31e"
+todolistid = "5f6076e68cc021208da06d2b"
+donelistid = "5f6076e96994a166c05385a6"
```

Comment:
Your README needs to be updated - it should contain everything you need to know to run the app after cloning the repository. So it needs to tell the reader to add Trello credentials etc.

Also, less importantly, it would be easier to vary these between developers/environments if they are also environment variables like the secrets.